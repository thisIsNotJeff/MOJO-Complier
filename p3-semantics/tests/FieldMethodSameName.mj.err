 line 1, column 19: error: field and method with the same name (f)
