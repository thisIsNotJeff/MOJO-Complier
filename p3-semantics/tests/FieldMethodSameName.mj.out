BEGIN _BUILTIN
_BUILTIN.int=int
_BUILTIN.bool=bool
_BUILTIN.false:bool=false
_BUILTIN.true:bool=true
_BUILTIN.Null=^null
_BUILTIN.nil:Null=0
_BUILTIN.Refany=^null
_BUILTIN.Address=^null
_BUILTIN.Root=null
object {
}
_BUILTIN.Text=^null
_BUILTIN.First:(...)
_BUILTIN.Last:(...)
_BUILTIN.New:(...)
_BUILTIN.Number:(...)
BEGIN FieldMethodSameName
FieldMethodSameName.T=Root
object {
0: f:int
0: f:()=null
}
END FieldMethodSameName
END _BUILTIN
